### PR TITLE
About Node version managers.

### PR DESCRIPTION
**Pull request by VarunPathak17**
## Summary of change

[Please briefly describe what you have changed.]

## Additional context

[If a screenshot would be useful (e.g. for UI changes), please provide one. If not, please remove this section]

## Related issue

Closes #[issue number]

## Pull request checklist

Please keep this checklist in & ensure you have done the following:

- [ ] Read, understood and adhered to our [contributing document](https://github.com/knjk04/book-project/blob/master/CONTRIBUTING.md).
  - [ ] Ensure that you were first assigned to a relevant issue before creating this pull request
  - [ ] Ensure code changes pass all tests

- [ ] Read, understood and adhered to our [style guide](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md). A lot of our code reviews are spent on ensuring compliance with our style guide, so it would save a lot of time if this was adhered to from the outset. 

- [ ] Filled in the summary, context (if applicable) and related issue section. Replace the square brackets and its placeholder content with your contents. For an example, see any merged in pull request
  - [ ] Included a screenshot(s) if a UI change was involved (it may look different on a reviewer's device)

- [ ] Created a branch that has a descriptive name (what your branch is for in a few words and includes the issue number at the end, e.g. `test-reading-goal-123`

- [ ] Set this pull request to 'draft' if you are still working on it

- [ ] Resolved any merge conflicts

For any of the optional checkboxes (e.g. the screenshots one), still check it if it does not apply.

### CODE REVIEW

**Comment by knjk04** [comment]
Hi @VarunPathak17, could you rephrase the wording to "You can use nvm if you want to manage multiple Node versions on your computer" where the word 'nvm' links to https://github.com/nvm-sh/nvm?

> - Node >= 10 for React

Also, could you move this to be a sub-bullet point underneath the above bullet point?

Thanks

**Comment by VarunPathak17** [comment]
I have changed it and pushed also.

**Comment by knjk04** [comment]
image

The new commit isn't showing

**Comment by knjk04** [comment]
@VarunPathak17 Do you have an update on this?
